hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/message/ErrorMessage.java

```diff
@@ -12,7 +12,7 @@
 	HL7Error_Msg_EncryptionError("TXFR029E", "Encryption protocols failed with remote facility."),
 	
 	HL7Error_Msg_InvalidMSHSegment("HNPS002E", "Invalid MSH segment format"),
-	HL7Error_Msg_UnknownReceivingApplication("HNPS005E", "Unknown receiving application"),
+	HL7Error_Msg_UnknownReceivingApplication("HNPS005E", "Unknown receiving application:"),
```

Comment:
I wonder if we should include placeholders in the String to make it obvious that we will be adding additional variables. Instead of just manually appending stuff in the code. This would also be more flexible if the text to be added was in the middle of the String. So try something like "Unknown receiving application: %s" which can be used with String.format().